charts/nextcloud/README.md

```diff
@@ -101,12 +102,12 @@ The following table lists the configurable parameters of the nextcloud chart and
 | `nextcloud.extraVolumes`                                     | specify additional volumes for the NextCloud pod        | `{}`                                        |
 | `nextcloud.extraVolumeMounts`                                | specify additional volume mounts for the NextCloud pod  | `{}`                                        |
 | `nextcloud.securityContext`                                  | Optional security context for the NextCloud container   | `nil`                                       |
-| `nextcloud.podSecurityContext`                               | specify additional volume mounts for the NextCloud pod  | `nil`                                       |
+| `nextcloud.podSecurityContext`                               | Optional security context for the NextCloud pod (applies to all containers in the pod) |         | `nil`                                       |
```

Comment:
Could reformat the table?

Reply:
No sorry, I meant that all the | should line up.

Reply:
Sorry for the confusion. All the information should stay there, just the table should be formatted so that the | all align in the markdown source.